### PR TITLE
Anglerfish

### PR DESCRIPTION
**Pull request by JohnCorby**
it pretty much works perfectly.
there might be a slight desync when someone joins, but it gets fixed when another state change happens, so no big deal.
please try it yourself, if you have time.
I think there's a small bug if you die because your character may(?) stick around which causes the fish to continue targeting you, but, again, it's small enough to not really be too consequential.

### CODE REVIEW

File: QSB/Anglerfish/Events/AnglerChangeStateEvent.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+using static AnglerfishController;
+using EventType = QSB.Events.EventType;
+
+namespace QSB.Anglerfish.Events
+{
+	public class AnglerChangeStateEvent : QSBEvent<AnglerChangeStateMessage>
+	{
+		public override EventType Type => EventType.AnglerChangeState;
+		public override void SetupListener() => GlobalMessenger<QSBAngler>.AddListener(EventNames.QSBAnglerChangeState, Handler);
+		public override void CloseListener() => GlobalMessenger<QSBAngler>.RemoveListener(EventNames.QSBAnglerChangeState, Handler);
```
Comment (misternebula) [inline QSB/Anglerfish/Events/AnglerChangeStateEvent.cs:15]: new lines between methods please

Comment (JohnCorby) [reply QSB/Anglerfish/Events/AnglerChangeStateEvent.cs:15]: done

File: AssetBundles/conversation.manifest [hunk 2]
```diff
@@ -40,5 +30,4 @@ ClassTypes:
 SerializeReferenceClassIdentifiers: []
 Assets:
 - Assets/DialogueBubble.prefab
-- Assets/GameAssets/Texture2D/UI_DialogueBox_d.png
```
Comment (misternebula) [inline AssetBundles/conversation.manifest:43]: ?

Comment (JohnCorby) [reply AssetBundles/conversation.manifest:43]: yeah i have no clue. unity just did that on it's own

**Review by misternebula — APPROVED** [review]

**Pull request merged**